acceptance/config/beaker/local/el6/1host.cfg

```diff
@@ -14,7 +14,7 @@ HOSTS:
     group: puppet
```

Comment:
@rlinehan Ideally, yes we could remove these, but beaker hasn't been updated yet and this is how they're handling the configuration values in `puppet`.